test_conformance/api/test_native_kernel.cpp

```diff
@@ -46,8 +46,9 @@ int test_native_kernel(cl_device_id device, cl_context context, cl_command_queue
     }
 
     clMemWrapper streams[ 2 ];
-#if !(defined (_WIN32) && defined (_MSC_VER))
-    cl_int inBuffer[ n_elems ], outBuffer[ n_elems ];
+#if !(defined(_WIN32) && defined(_MSC_VER))
```

Comment:
It's probably better to get rid of the entire `#if`/`#else` block and drop the non-standard `_malloca`, and use `std::vector` instead.  That way we won't need to worry about freeing the memory using `delete[]` either.

Reply:
Thanks. Updated.

Reply:
Thanks!